src/status_im/ui/screens/wallet/send/styles.cljs

```diff
@@ -30,4 +30,71 @@
    :height 24})
 
 (def toolbar-title-icon
-  (merge toolbar-icon {:opacity 0.4}))
\ No newline at end of file
+  (merge toolbar-icon {:opacity 0.4}))
+
+(def toolbar-buttons-container
+  {:flex-direction  :row
+   :flex-shrink     1
+   :justify-content :space-between
+   :width           68
+   :margin-right    12})
+
+(def choose-recipient-container
+  {:flex-direction  :row
+   :padding-top     20
+   :padding-bottom  20
+   :justify-content :center})
+
+(def choose-recipient-label
+  {:color :white})
+
+(def recipient-buttons
+  {:flex-direction    :column
+   :margin-horizontal 28
+   :margin-vertical   10
+   :border-radius     5
+   :background-color  styles/color-blue5})
+
+(def recipient-icon {:margin-right 20})
+
+(def recipient-button
+  {:flex-direction  :row
+   :justify-content :space-between
+   :margin-vertical 10
+   :margin-left     20})
+
+(def recipient-button-text
+  {:color     :white
+   :font-size 17})
+
+(defn recipient-touchable [divider?]
+  (cond-> {:border-color styles/color-gray-transparent-medium-light}
+    divider? (assoc :border-bottom-width 1)))
+
+(def qr-container
+  {:flex              1
+   :margin-horizontal 32})
+
+(def preview {:flex            1
```

Comment:
indentation

Reply:
Isn't this correctly indented?

Reply:
map should be on the new line, like above